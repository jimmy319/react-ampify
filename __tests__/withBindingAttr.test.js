const reactAmpify = require('../index.js')
const React = require('react')

test('withBindingAttrs functionality', () => {
  // arrangement
  const { withBindingAttrs } = reactAmpify

  // action
  const expected = withBindingAttrs(<input className='test' />, {value: 'data.value'})

  // assertion
  expect(expected).toMatchSnapshot()
})
