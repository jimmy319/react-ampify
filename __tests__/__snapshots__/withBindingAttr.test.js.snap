// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`withBindingAttrs functionality 1`] = `
<input
  [value]="data.value"
  className="test"
/>
`;
